src/AvionicsCalculations.cpp

```diff
@@ -8,6 +8,9 @@
 bool Avionics::processData() {
   bool success = true;
 
+  uint32_t timeSinceLaunch; // need function to calculate this
```

Comment:
@johnldean iirc Lasagna can detect launch - since you're familiar with that + with the interpolated GPS_TIME would you mind coding this in? It should just give the current time relative to launch time in seconds.